guacamole-docker/bin/start.sh

```diff
@@ -427,15 +427,15 @@ fi
 # Validate that at least one authentication backend is installed
 #
 
-if [ -z "$INSTALLED_AUTH" ]; then
+if [ -z "$INSTALLED_AUTH" -a -z "$GUACAMOLE_HOME_TEMPLATE" ]; then
```

Comment:
Nope. The sanity check should fail only if both `$INSTALLED_AUTH` *and* `$GUACAMOLE_HOME_TEMPLATE` are blank. If either is defined, then the sanity check should pass.